cli/src/cli_util.rs

```diff
@@ -1157,6 +1158,17 @@ See https://github.com/martinvonz/jj/blob/main/docs/working-copy.md#stale-workin
         if let Some(stats) = stats {
             print_checkout_stats(ui, stats, new_commit)?;
         }
+        if !self.global_args.quiet && Some(new_commit) != maybe_old_commit {
+            let mut formatter = ui.stderr_formatter();
+            let conflicts = new_commit.tree()?.conflicts().collect_vec();
+            if !conflicts.is_empty() {
+                writeln!(
+                    formatter.labeled("conflict"),
+                    "There are unresolved conflicts at these paths:"
```

Comment:
nit: just leave the message unlabeled? labeling it as `"conflict"` (of `jj log`) seems odd.